src/PlaywrightSharp/JSHandle.cs

```diff
@@ -19,6 +19,19 @@ internal JSHandle(ExecutionContext context, IRemoteObject remoteObject)
 
         internal IRemoteObject RemoteObject { get; set; }
 
+        /// <inheritdoc cref="IJSHandle.EvaluateAsync{T}(string, object[])"/>
+        public Task<T> EvaluateAsync<T>(string pageFunction, params object[] args)
+        {
+            object[] newArgs = new object[args.Length + 1];
+            newArgs[0] = this;
+            args.CopyTo(newArgs, 1);

```

Comment:
InsertAt.

Reply:
I don't agree with this. we know in advance the size of the array, there's no need to create a list and convert back to an array

Reply:
Agreed, in the previous comment, I mentioned moving this recipe to the InsertAt extension method. We could rename it to `Prepend`.